exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/Checker.java

```diff
@@ -20,11 +20,24 @@
 import org.apache.calcite.sql.SqlCallBinding;
 import org.apache.calcite.sql.SqlOperandCountRange;
 import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.type.SqlOperandCountRanges;
 import org.apache.calcite.sql.type.SqlOperandTypeChecker;
 
 class Checker implements SqlOperandTypeChecker {
```

Comment:
Looks like we are creating instances of this class unnecessarily (one per UDF).

Can we have static instances for some counts (0,1,2,3, any)?